index.php

```diff
@@ -1,4 +1,13 @@
 <?php
+    /**
+     * Uncomment the next lines for debug.
+     * Don forget to reset them when checking in
```

Comment:
Issue right here, can we add an environment key to our .env.example and toggle based on that? No dirty in comment and no getting this into git